pkg/providers/nutanix/config/cp-template.yaml

```
@@ -177,11 +177,26 @@ spec:
       memorySize: {{.memorySize}}
       systemDiskSize: {{.systemDiskSize}}
       image:
+{{- if (eq .imageIDType "name") }}
         type: name
         name: "{{.imageName}}"
+{{ else if (eq .imageIDType "uuid") }}
+        type: uuid
+        name: "{{.imageUUID}}"
+{{ end }}
       cluster:
+{{- if (eq .imageIDType "name") }}
```

This should use `nutanixPEClusterIDType` not `imageIDType`
```suggestion
{{- if (eq .nutanixPEClusterIDType "name") }}
```